.github/workflows/start-release.yml

```
@@ -0,0 +1,53 @@
+name: Trigger Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release-setup:
+    name: Setup Release Branch
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+      id: checkout
+      with:
+        submodules: "recursive"
+        ref: "develop"
+    - name: Pull Tags
+      id: pull-tags
+      # We don't want a full heavyweight checkout with full history
+      # checkout action only can get tags + full history, so do this separately.
+      # Don't need tags in submodules.
+      run: git fetch --no-recurse-submodules origin 'refs/tags/*:refs/tags/*'
+    - name: Get latest version
+      id: latest-version
+      run: |
+        LATEST_TAG=$(git tag | sort -V | tail -n 1 | cut -c 2- )
```

```suggestion
        LATEST_TAG=$(git tag --list "v*.*.*" | sort -V | tail -n 1 | cut -c 2- )
```

And we should set branch protection rules for `v*.*.*` on the github UI so we cannot accidentally mess up the versioning 👍